component-samples/components-oauth/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>OAuth with ArcGIS Map Components</title>
+    <link rel="icon" href="data:;base64,=" />
+  </head>
+
+  <body>
+    <arcgis-map item-id="d5dda743788a4b0688fe48f43ae7beb9">
```

Comment:
Since they need to fill in their portal url, should they also need to set their own item id?